app/main/posts/detail/detail.html

```diff
@@ -155,5 +155,6 @@ <h3 class="listing-heading" translate="app.tasks">Tasks</h3>
 
     <post-add-form post="post" ng-if="!post.form && post.allowed_privileges.indexOf('update') !== -1"></post-add-form>
     <post-messages post="post" ng-if="post.contact.id"></post-messages>
+    <ush-logo></ush-logo>
```

Comment:
Does this just float on every view when embedded? Is it possible to just add it once to index.html?